Automattic/woocommerce-payments
# Updates the version of subscriptions-core bundled in WooPayments to 6.7.1

#### Changes proposed in this Pull Request









This PR bumps the version of subscriptions-core bundled in WooPayments from 6.6.0 to 6.7.1. Here are the changes coming with this version bump that are relevant to merchants still using the WCPay Subscriptions feature:

```
= 6.7.1 - 2024-01-17 =
* Fix - Resolved an error that would occur with WC 8.5.0 when editing a subscription customer from the admin dashboard.

= 6.7.0 - 2024-01-11 =
* Update - Changed the edit subscription product "Expire after" (Subscription length) so it more clearly describes when a subscription will automatically stop renewing.
* Fix - Resolved an issue that caused ordering the Admin Subscriptions List Table to not work when HPOS is enabled.
```

#### Testing instructions





1. Checkout this branch
2. Run composer install
3. Confirm `vendor/woocommerce/subscriptions-core` is running the latest

-------------------
